packages/xo-web/src/xo-app/vm/tab-general.js

```diff
@@ -48,13 +48,18 @@ export default connectStore(() => {
 
   return {
     lastShutdownTime: createGetVmLastShutdownTime(),
+    tasks: createGetObjectsOfType('task')
+      .pick((_, { vm }) => Object.keys(vm.current_operations))
```

Comment:
Unfortunately, that's not good enough, because the selector passed to `pick()` will change at each call (new array) which will trigger a new pick each time.

Reply:
You need to create a nested selector watching `vm.current_operations`.

Reply:
FYI, it was the same issue before: you created a new predicate each time.